walkers/src/extras/places.rs

```diff
@@ -98,9 +104,15 @@ impl Places {
 }
 
 impl Plugin for Places {
-    fn draw(&self, response: &Response, painter: Painter, projector: &crate::Projector) {
+    fn draw(
+        &self,
+        response: &Response,
+        painter: Painter,
+        projector: &crate::Projector,
+        gesture_handled: bool,
```

Comment:
Or use Projector to store this flag? Any suggestions?

Reply:
Nah, `Projector` should only do the math. I was thinking about aggregating all of these parameters, but still can't decide if it's a good idea.